app/src/main/java/framgia/com/ichat/screen/home/HomePresenter.java

```diff
@@ -0,0 +1,24 @@
+package framgia.com.ichat.screen.home;
+
+import framgia.com.ichat.data.model.User;
+import framgia.com.ichat.data.repository.HomeRepository;
+
+public class HomePresenter implements HomeContract.Presenter {
+    private HomeContract.View mView;
+    private HomeRepository mRepository;
+
+    public HomePresenter(HomeRepository repository) {
+        mRepository = repository;
+    }
+
+    @Override
+    public User getCurrentUser() {
+        return mRepository.getCurrentUser();
+
+    }
+
+    @Override
+    public void setView(HomeContract.View view) {
+        this.mView = view;
```

Comment:
remove this.
this chỉ sử dụng để phân biệt khi biến global và value có cùng tên thôi e